apps/run.go

```diff
@@ -322,42 +322,43 @@ func (runCtx *runContext) connectToRunServer(ctx context.Context) (*http.Respons
 		return nil, nil, errgo.New("No attach URL to connect to")
 	}
 
-	req, err := http.NewRequest("CONNECT", runCtx.attachURL, nil)
+	url, err := url.Parse(runCtx.attachURL)
 	if err != nil {
 		return nil, nil, errgo.Mask(err, errgo.Any)
 	}
-	token, err := runCtx.scalingoClient.GetAccessToken(ctx)
 
+	// Establish an initial TCP connection
+	conn, err := net.Dial("tcp", url.Host)
 	if err != nil {
-		return nil, nil, errgo.Notef(err, "fail to generate auth")
+		return nil, nil, errgo.Mask(err, errgo.Any)
```

Comment:
issue: we are in the process of removing `errgo` from our codebase. When you see a use of `errgo`, please replace it with the use of `github.com/Scalingo/go-utils/errors/v2` (https://www.notion.so/scalingooriginal/Go-Error-Management-1ca42114c730415e95755027935eaccc)
```suggestion
		return nil, nil, errors.Wrap(ctx, err, "establish initial TCP connection")
```

Reply:
I'll refactor run.go and submit a proper commit for this